pino.js

```diff
@@ -217,7 +224,7 @@ function fastRep (s) {
   } else {
     result += str.slice(last)
   }
-  return result
+  return point < 32 ? JSON.stringify(str) : '"' + result + '"'
```

Comment:
You are not correct. If a code point below 32 is found, it becomes `JSON.stringify(str)`, which will yield the correct result.

The unit tests to be compliant with the spec are passing.